README.md

```diff
@@ -85,7 +85,7 @@ $ brew install carthage
 To integrate Alamofire into your Xcode project using Carthage, specify it in your `Cartfile`:
 
 ```ogdl
-github "Alamofire/Alamofire" "swift-2.0"
+github "Alamofire/Alamofire" ~> 2.0
```

Comment:
Even if it is `2.0` for CocoaPods? (see https://github.com/Alamofire/Alamofire/commit/343e24073990172de3c55871762f68d49838a3ce)

Does Carthage handle this in a different way?